src/CMakeLists.txt

```diff
@@ -637,9 +637,23 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+
+set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
+if(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
@tchaikov 
"Fun" part is that I've already have line 30 with "matches" as well, in another WIP.
That got deleted when we removed the debug options from the Clang commit.
So I actually copied it from there. :)
But I'll make it consistent.